src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,52 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+  let arrayOfKeys = Object.keys(obj);
+  arrayOfKeys = arrayOfKeys.map((each) => parseInt(each, 10));
+  arrayOfKeys = arrayOfKeys.sort((a, b) => a - b);
+  arrayOfKeys = arrayOfKeys.map((each) => each.toString());
```

Comment:
You could chain these operations.

Also, you can sort strings, no need to `parseInt`, sort, and turn back to a string. `Object.keys().sort()` would work just as well.

Reply:
Sorting with the keys as strings didn't achieve quite the same result as converting them to numbers, as all values starting with 1 came before values starting with 2 regardless of which was bigger. Using the `parseInt` before sorting worked better in this case.

Reply:
You're right.